content/docs/develop/operate/upgrade/index.md

```diff
@@ -7,17 +7,79 @@ keywords: upgrade
 
 ## Upgrade from 0.6.2 to 0.7
 
-* To download and install the Keptn CLI for version 0.7.0, please refer to the [Install Keptn CLI section](../setup-keptn/#install-keptn-cli).
+1. To download and install the Keptn CLI for version 0.7.0, please refer to the [Install Keptn CLI section](../setup-keptn/#install-keptn-cli).
 
-* To upgrade your Keptn installation from 0.6.2 to 0.7.0, you can deploy a *Kubernetes Job* that will take care of updating all components to the 0.7.0 release. Please [verify that you are connected to the correct Kubernetes cluster](../../reference/troubleshooting/#verify-kubernetes-context-with-keptn-installation)
-before deploying the upgrading job with the next command:
+1. To upgrade your Keptn installation from 0.6.2 to 0.7.0, a *Kubernetes Job* is provided that upgrades all components to the 0.7.0 release. 
+
+    * Please [verify that you are connected to the correct Kubernetes cluster](../../reference/troubleshooting/#verify-kubernetes-context-with-keptn-installation)
+before deploying the upgrading job.
+
+    * Keptn 0.7 uses Helm 3 while previous Keptn releases rely on Helm 2. This means that you have to upgrade the Helm releases of your Keptn-managed services. Otherwise, a `keptn send new-artifact` does not work anymore. For upgrading Helm releases, two options are availble as outlined below. Please take into account that the end-of-life period of Helm 2 begins on [August 13th, 2020](https://helm.sh/blog/covid-19-extending-helm-v2-bug-fixes/).
+
+### Job without Helm 3 Upgrade
+
+:mag: **Info:** By using this upgrader job, Helm releases are **not** converted to Helm 3.0 and still on version Helm 2. After executing the upgrader job, you need to manually convert the Helm releases on your Kubernetes cluster as explained below.
 
 ```console
 kubectl delete job upgrader -n default
 kubectl apply -f https://raw.githubusercontent.com/keptn/keptn/0.7.0/upgrader/upgrade-062-070/upgrade-job.yaml
 ```
 
-* To check the status of the update job, please execute:
+**Manual converting Helm release from Helm 2 to 3:**
+
+* Install Helm v2.14.3 CLI (*Note*: this CLI version is called to `helm`):
+```console
```

Comment:
I have one additional comment on the whole installation of Helm CLI... What you outlined here should work for Linux and I guess MacOS, but it won't work on Windows and also not with Powershell.

So I think we need to clarify that this only works with Linux and MacOS.

For Windows, an easy way to overcome this would be to start a Docker Container on the Kubernetes cluster with kubectl, helm2 and helm3 pre-installed.

Reply:
I added a note for Windows users.